packages/codemodel.go/src/gocodemodel.ts

```diff
@@ -837,6 +837,8 @@ export interface TimeType extends QualifiedType {
   packageName: 'time';
 
   dateTimeFormat: DateTimeFormat;
+
+  utc: boolean;
```

Comment:
This just wires in the data. Complete UTC time handling is tracked in https://github.com/Azure/autorest.go/issues/942.